scripts/monitoring/cron-send-create-app.py

```diff
@@ -167,9 +169,12 @@ def setup(config):
     except Exception:
         pass # don't want exception if project not found
 
+    # Create a new project using 'oc' instead of 'oc adm'.
+    # This will test the project-request template as part of project creation.
```

Comment:
This will only test `project-request template` on the first time creation, as it does not remove the project at the end.

Reply:
We cannot remove the project as it is possible multiple tests are ongoing (app create and app build) and removing the project will interfere with the other execution.

Reply:
Good catch. I thought it would be ok to just test the app itself, but you're right, we need a new project for each app-create run. This seems doable. I can work on it.

Reply:
Done. It creates a project each time now.